src/component-library/tailwind.css

```diff
@@ -7,4 +7,70 @@
         font-size: 16px;
         /* font-family: monospace; */
     }
-}
\ No newline at end of file
+}
+
+/*@layer base {*/
```

Comment:
@MytsV any reason to keep these commented css styles here?

Reply:
Are storybooks able to pick up this css file?

Reply:
Sorry, that's still a WIP and something got left from connecting shadcn/ui. I'll review it when working on the toasts in this PR.

> @MytsV any reason to keep these commented css styles here?

Reply:
> Are storybooks able to pick up the ag-grid-rucio css file?

Yes, and, luckily, without any additional setup!
![image](https://github.com/user-attachments/assets/70eff726-ab65-441d-95a9-c144f23ffa97)